src/components/Navigation/Header.vue

```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
```

Comment:
```suggestion
            class="mr-1 mt-sm-1 teal darken-2 pl-2"
```
Reduce padding left before the icon to harmonize the button

Reply:
added